backend/src/nodes/impl/external_stable_diffusion.py

```diff
@@ -120,8 +120,8 @@ def post(path, json_data: Dict) -> Dict:
     return response.json()
 
 
-def nearest_valid_size(width, height):
-    return (width // 8) * 8, (height // 8) * 8
+def nearest_valid_size(width, height, step=8):
```

Comment:
Since non-external-SD nodes are now using this mehod, it might be good to make this a util.util method.